.semaphore/semaphore.yml

```diff
@@ -97,9 +97,9 @@ blocks:
         - name: 'Build and push artifacts'
           commands:
             - wget https://dot.net/v1/dotnet-install.ps1 -OutFile dotnet-install.ps1
-            - powershell -ExecutionPolicy ByPass -File dotnet-install.ps1 -Version 6.0.403 -InstallDir C:\dotnet
+            - powershell -ExecutionPolicy ByPass -File dotnet-install.ps1 -Version 8.0.300 -InstallDir C:\dotnet
```

Comment:
8.0.401 is the latest SDK version. Is there a reason this one is only using 300?